witness/src/Dscp/Snowdrop/Validators.hs

```diff
@@ -6,16 +6,14 @@ module Dscp.Snowdrop.Validators
     ) where
 
 
-import Data.Default (def)
-import qualified Data.Map as M
+import qualified Snowdrop.Block as SD
 import qualified Snowdrop.Core as SD
-import qualified Snowdrop.Model.Block as SD
 import qualified Snowdrop.Util as SD
 
 import Dscp.Core hiding (PublicationTxWitness)
-import Dscp.Crypto (PublicKey, Signature, hash, verify)
+import Dscp.Crypto (PublicKey, hash, verify)
 import Dscp.Snowdrop.AccountValidation
-import Dscp.Snowdrop.Configuration
+import Dscp.Snowdrop.Configuration hiding (PublicKey)
```

Comment:
`Dscp.Snowdrop.Configuration` doesn't export `PublicKey`. If it does, that's a mistake.